src/response/response.ts

```diff
@@ -0,0 +1,45 @@
+import { MessageResult } from './message-result';
+
+type ResponseStatus = {
+  code: number
+  message: string
+};
+
+type ResponseOpts = {
+  status?: ResponseStatus
+  results?: MessageResult[];
+};
+
+/**
+ * class used to create a response that will be returned for a given request
+ */
+export class Response {
+  // present ONLY if there is a general request-related issue
+  // e.g. malformed request, invalid target. `status` and `replies` are mutually exclusive
```

Comment:
> We changed to "results"

Oops, good catch! it is `replies` in the spec. @csuwildcat @thehenrytsai which do y'all want to use, `results` or `replies`?

<img width="1188" alt="image" src="https://user-images.githubusercontent.com/4887440/176050272-c07a6822-b275-4132-bc80-130dbb0dba09.png">


> Also spec really says mutually exclusive? What's the reason against an explicit 200? bloat?

Yep, that's how i read it:
<img width="1064" alt="image" src="https://user-images.githubusercontent.com/4887440/176050383-a4519839-23da-4dd9-837d-aff80a2f5695.png">

@csuwildcat thoughts? what was your thinking behind making `replies` and `status` mutually exclusive? if we are going to keep them mutually exclusive, and `status` can only show up if there's an error, it may make more sense to just name it `error` instead of `status`?

Reply:
1. I think message<->reply make more sense than message<->result.
1. Agree with you that if it's meant to be mutually exclusive then `error` makes more sense!

Reply:
@thehenrytsai `MessageResult` -> `MessageReply` resolved: https://github.com/TBD54566975/dwn-sdk-js/commit/5f47f62123688769db41aacaef7923039f137e1e